src/mixins/_fluid.scss

```diff
@@ -108,6 +108,23 @@
   }
 }
 
+@mixin margin-inline($min-width, $max-width, $min, $max, $include-min: true) {
+  @if $include-min {
+    margin-inline-end: $min;
+    margin-inline-start: $min;
+  }
+
+  @media (min-width: $min-width) {
+    margin-inline-end: fluid-calc($min-width, $max-width, $min, $max);
+    margin-inline-start: fluid-calc($min-width, $max-width, $min, $max);
+  }
+
+  @media (min-width: $max-width) {
+    margin-inline-end: $max;
+    margin-inline-start: $max;
+  }
+}
```

Comment:
> can you highlight the sections that are most confusing?

Some extra comments here would be helpful. The rest is okay without more comments. 👍 

Thanks, @tylersticka!

Reply:
Gotcha. Since this mixin is almost exactly the same as all the others in this file, I might check to see if any of the others are already commented first before singling this one out. (I wish Sass let you use string interpolation on property names, then we wouldn't need property-specific versions of these mixins.)

Reply:
@gerardo-rodriguez I unfortunately don't think that I have any additional comments to provide other than those at the top of the file: https://github.com/cloudfour/cloudfour.com-patterns/blob/a2d9e84b77619bfd71cc7cdd946c4a2769141990/src/mixins/_fluid.scss#L3-L14

Reply:
Thanks, @tylersticka, I totally missed this comment! 👍